modules/compliance-custom-node-pools.adoc

```diff
@@ -0,0 +1,94 @@
+
+If your cluster uses custom node pools outside the default `worker` and `master` node pools, you will need to supply additional variables to ensure the Compliance Operator aggregates a configuration file for that node pool.
+
+.Procedure
+
+. To check the configuration against all pools in an example cluster containing `master`, `worker`, and custom `example` node pools, set the value of the `ocp-var-role-master` and `opc-var-role-worker` fields to `example` in the `TailoredProfile` object:
++
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: TailoredProfile
+metadata:
+  name: cis-example-tp
+spec:
+  extends: ocp4-cis
+  title: My modified NIST profile to scan example nodes
+  setValues:
+  - name: ocp4-var-role-master
+    value: example
+    rationale: test for example nodes
+  - name: ocp4-var-role-worker
+    value: example
+    rationale: test for example nodes
+  description: cis-example-scan
+----
+
+. Add the `example` role to the `ScanSetting` object that will be stored in the `ScanSettingBinding` CR:
++
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: ScanSetting
+metadata:
+  name: default
+  namespace: openshift-compliance
+rawResultStorage:
+  rotation: 3
+  size: 1Gi
+roles:
+- worker
+- master
+- example
+scanTolerations:
+- effect: NoSchedule
+  key: node-role.kubernetes.io/master
+  operator: Exists
+schedule: '0 1 * * *'
+----
+
+. Create a scan using `ScanSettingBinding` CR:
++
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: ScanSettingBinding
+metadata:
+  name: cis
+  namespace: openshift-compliance
+profiles:
+- apiGroup: compliance.openshift.io/v1alpha1
+  kind: Profile
+  name: ocp4-cis
+- apiGroup: compliance.openshift.io/v1alpha1
+  kind: Profile
+  name: ocp4-cis-node
+- apiGroup: compliance.openshift.io/v1alpha1
+  kind: TailoredProfile
+  name: cis-example-tp
+settingsRef:
+  apiGroup: compliance.openshift.io/v1alpha1
+  kind: ScanSetting
+  name: default
+----
+
+The Compliance Operator checks the runtime `KubeletConfig` through the `Node/Proxy` API object, and then uses variables such as `ocp-var-role-master` and `ocp-var-role-worker` to determine the nodes it performs the check against. In the `ComplianceCheckResult`, the `KubeletConfig` rules will be shown as `ocp4-cis-kubelet-*`. The scan only passes if all selected nodes pass this check.
+
+.Verification
+
+* The Platform KubeletConfig rules are checked through the `Node/Proxy` object. You can find those rules by running the following command:
++
+[source,terminal]
+----
+$ oc get rules -o json | jq '.items[] | select(.checkType == "Platform") | select(.metadata.name | contains("ocp4-kubelet-")) | .metadata.name'
```

Comment:
> Do not use jq in commands (unless it is truly required), because this requires users to install the jq tool.  Oftentimes, the same or similar result can be accomplished using `jsonpath` for `oc` commands.

This is out of scope for a merge review, but consider fixing it in the future.